protocol/v2/types/ssvshare.go

```diff
@@ -103,4 +112,5 @@ type Metadata struct {
 	BeaconMetadata *beaconprotocol.ValidatorMetadata
 	OwnerAddress   common.Address
 	Liquidated     bool
+	Invalid        bool
```

Comment:
maybe we can rename this to `InvalidEncryptedShare` to be more self-explanatory?